pkgs/applications/virtualization/docker/default.nix

```
@@ -28,7 +29,8 @@ rec {
     });
 
     docker-containerd = (containerd.override { inherit go; }).overrideAttrs (oldAttrs: rec {
-      name = "docker-containerd";
+      name = "docker-containerd-${version}";
```

does this mean that the abstraction breaks down on override?
```suggestion
      pname = "docker-containerd";
```